patches/server/0819-Add-config-option-for-logging-player-ip-addresses.patch

```diff
@@ -93,12 +93,3 @@ index 99aa8f2ba2f10578f37de621d1a5a8e222cd70b1..eaa005c1c9b4386bcdbe1d6eb28c3eca
          }
  
          // Spigot start - spawn location event
```

Comment:
Am I being blind or did you just entirely break another patch?

Reply:
s1 uses the exact same code, that patch duplicated the same code twice

Reply:
That patch also includes:
```java
         if (connection.getRemoteAddress() != null) {
-            s1 = connection.getRemoteAddress().toString();
+            s1 = com.destroystokyo.paper.PaperConfig.logPlayerIpAddresses ? connection.getRemoteAddress().toString() : "<ip address withheld>"; // Paper
         }
```